packages/react/src/create-strict-api/__tests__/index.test.tsx

```diff
@@ -1,24 +1,31 @@
 /** @jsxImportSource @compiled/react */
 import { render } from '@testing-library/react';
 
-import { css, cssMap, XCSSProp } from './__fixtures__/strict-api';
+import { css, cssMap, XCSSProp, cx } from './__fixtures__/strict-api';
 
 describe('createStrictAPI()', () => {
   describe('css()', () => {
     it('should type error when circumventing the excess property check', () => {
-      const styles = css({
+      const stylesOne = css({
```

Comment:
I've split this into two because the entire object is type checked now, meaning it hits the first type error and then stops trying to typecheck the rest. Type safety has not been lost but because of the type refactor behaves slightly differently.